src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/Templates/DbContext/NewDbContextFactory.cshtml

```diff
@@ -0,0 +1,25 @@
+@inherits Microsoft.VisualStudio.Web.CodeGeneration.Templating.RazorTemplateBase
+@using System.Collections.Generic;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+@{
+    if (!Model.DbContextNamespace.Equals(Model.ProgramNamespace, StringComparison.Ordinal))
+    {
+@:using @Model.ProgramNamespace;
+    }
+}
+@{
+    var factoryClassName = @Model.DbContextTypeName+"Factory";
+}
+namespace @Model.DbContextNamespace
+{
+    public class @factoryClassName : IDbContextFactory<@Model.DbContextTypeName>
+    {
+        public @Model.DbContextTypeName Create(DbContextFactoryOptions options) =>
```

Comment:
This should be `string[] args` and passed into `.BuildWebHost`. See dotnet/templating#705